From eric.y.miao@gmail.com  Tue Jan  5 14:23:45 2010
From: Eric Miao <eric.y.miao@gmail.com>
Date: Fri, 1 Jan 2010 15:43:28 +0800
Subject: driver core: make platform_device_id table const
To: Kay Sievers <kay.sievers@vrfy.org>, Greg KH <gregkh@suse.de>
Message-ID: <f17812d70912312343x31656af6sdb4f3556e7022c05@mail.gmail.com>


The platform ID table is normally const, force that by adding the attribute.

Signed-off-by: Eric Miao <eric.y.miao@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/platform_device.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/platform_device.h
+++ b/include/linux/platform_device.h
@@ -21,7 +21,7 @@ struct platform_device {
 	u32		num_resources;
 	struct resource	* resource;
 
-	struct platform_device_id	*id_entry;
+	const struct platform_device_id	*id_entry;
 
 	/* arch specific additions */
 	struct pdev_archdata	archdata;
